File: src/algorithms/maximum-count.js
/**
 * Given an array nums sorted in increasing order, returns the number of 
 * positive integers if there are more positive numbers than negative, otherwise
 * returns the number of negative numbers.
 * 
 * @param {number[]} nums - An array of numbers
 * @returns {number} - Returns the max number between the number of positive
 *     numbers and negative numbers
 */
function maximumCount (nums) {
  let pos = 0;
  let neg = 0;
  for (const num of nums) {
    if (num > 0) {
      pos++;
    } else if (num < 0) {
      neg++;
    }
  }
  return Math.max(pos, neg);
}

File: src/algorithms/maximum-count.py
def maximum_count(nums):
    """Given an array nums sorted in non-decreasing order, return the maximum
    between the number of positive integers and the number of negative integers.
    In other words, if the number of positive integers in nums is pos and the
    number of negative integers is neg, then return the maximum of pos and neg.

    Args:
        nums (List[int]): A list of integers.

    Returns:
        Integer: Returns the maximum number of positive or negative integers.
    """
    pos = 0
    neg = 0
    for num in nums:
        if num > 0:
            pos += 1
        elif num < 0:
            neg += 1
    return max(pos, neg)
